tests/FeeSharingCollectorMultiToken.test.js

```diff
@@ -0,0 +1,3051 @@
+        before(async () => {
+            await loadFixture(protocolDeploymentFixture);
+        });
+        beforeEach(async () => {
+            snapshot = await takeSnapshot();
+        });
+        afterEach(async () => {
+            await snapshot.restore();
+        });
+
+        it("Check owner & implementation", async () => {
+            const proxyOwner = await feeSharingCollectorProxyObj.getProxyOwner();
+            const implementation = await feeSharingCollectorProxyObj.getImplementation();
+
+            expect(implementation).to.be.equal(feeSharingCollectorLogic.address);
+            expect(proxyOwner).to.be.equal(root);
+        });
+
+        it("Set new implementation", async () => {
+            const newFeeSharingCollector = await FeeSharingCollector.new();
+            await feeSharingCollectorProxyObj.setImplementation(newFeeSharingCollector.address);
+            const newImplementation = await feeSharingCollectorProxyObj.getImplementation();
+
+            expect(newImplementation).to.be.equal(newFeeSharingCollector.address);
+        });
+    });
+
+    describe("withdrawFees", () => {
+        it("Shouldn't be able to use zero token address", async () => {
+            await protocolDeploymentFixture();
+            await expectRevert(
+                feeSharingCollector.withdrawFees([ZERO_ADDRESS]),
+                "FeeSharingCollectorMultiToken::withdrawFees: token is not a contract"
+            );
+        });
+
+        it("Shouldn't be able to withdraw if wRBTC loan pool does not exist", async () => {
+            await protocolDeploymentFixture();
+            // Unset the loanPool for wRBTC
+            await sovryn.setLoanPool([loanWrappedNativeToken.address], [ZERO_ADDRESS]);
+
+            //mock data
+            let lendingFeeTokensHeld = new BN(wei("1", "ether"));
+            let tradingFeeTokensHeld = new BN(wei("2", "ether"));
+            let borrowingFeeTokensHeld = new BN(wei("3", "ether"));
+            let totalFeeTokensHeld = lendingFeeTokensHeld
+                .add(tradingFeeTokensHeld)
+                .add(borrowingFeeTokensHeld);
+            let feeAmount = await setFeeTokensHeld(
+                lendingFeeTokensHeld,
+                tradingFeeTokensHeld,
+                borrowingFeeTokensHeld,
+                true
+            );
+
+            await expectRevert(
+                feeSharingCollector.withdrawFees([WrappedNativeToken.address]),
+                "FeeSharingCollectorMultiToken::withdrawFees: loan wrappedNativeTokenAddress not found"
+            );
+        });
+
+        it("Shouldn't be able to withdraw zero amount", async () => {
+            await protocolDeploymentFixture();
+            const tx = await feeSharingCollector.withdrawFees([SUSD.address]);
+            expectEvent(tx, "FeeWithdrawn", {
+                sender: root,
+                token: loanWrappedNativeToken.address,
+                amount: new BN(0),
+            });
```

Comment:
@cwsnt this test seems weird: shouldn't we have an "expectRevert" checking here, instead of an event of a successful tx?